plugins/matplotlib/express.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const app = express();
+const portNumber = 3000;
+const sourceDir = 'dist';
+
+app.use(express.static(sourceDir));
+
+app.listen(portNumber, () => {
+  console.log(`Express web server started: http://localhost:${portNumber}`);
+  console.log(`Serving content from /${sourceDir}/`);
+});
```

Comment:
What is this used for? Not seeing anywhere that it's used

Reply:
Removed